calliope/example_models/urban_scale/scenarios.yaml

```diff
@@ -24,6 +25,9 @@ overrides:
                         energy_cap: 35
                         purchase: 2000
             # boiler-end
+            heat_pipes:
```

Comment:
why is this added here? it seems like in the tests you are also and separately adding in binary_operation: True with direct overrides

Reply:
Ok, we'll keep in mind "cleaning up tests" for some undetermined future date!